.circleci/config.yml

```diff
@@ -29,6 +29,12 @@ jobs:
       - run:
           name: Build the package (includes running the tests)
           command: nix-build
+      - run:
+          name: Check formatting
+          command: |
+            result/bin/ormolu --mode inplace $(find src -type f -name "*.hs")
+            result/bin/ormolu --mode inplace $(find tests -type f -name "*.hs")
```

Comment:
This would probably match test fixtures, which are not formatted. We should probably only apply it to `*-out.hs` files. Or we can just remove this line because tests themselves ensure that `-out.hs` files are formatted correctly.

Reply:
Fixtures are in `data/`.

Reply:
Oh, yes, my mistake, sorry.